charthelp.py

```diff
@@ -12,12 +12,12 @@ def create_help(toolbar):
     helpitem.add_section(_('Basic usage'))
     helpitem.add_paragraph(_('First select data type:'))
     helpitem.add_paragraph(_('The free space in the Journal;'),
-            'import-freespace')
+                           'import-freespace')
     helpitem.add_paragraph(_('The types of Sugar Activities you have used;'),
-            'import-journal')
+                           'import-journal')
     helpitem.add_paragraph(_('The types of blocks used in Turtle Art.'),
-            'import-turtle')
-    helpitem.add_paragraph(_('The graph title is the same as the Activity title'))
```

Comment:
Wording change doesn't seem right; "is the same" seems okay to me.

Reply:
Done in 14f0084